openmdao/components/test/test_meta_model.py

```diff
@@ -352,6 +352,11 @@ def test_derivatives(self):
         prob['x'] = 0.125
         prob.run()
 
+        Jf = prob.calc_gradient(['x'], ['meta.f'], mode='fwd')[0][0]
+        Jr = prob.calc_gradient(['x'], ['meta.f'], mode='rev')[0]
+
+        assert_rel_error(self, Jf-Jr, 0.0, 1.0e-8)
```

Comment:
The lines below it test both of those values vs Finite Difference, which is internally consistent. I could comment out the assert, since the important part to me was adding the calc_gradient to isolate some keyerrors for debugging.